docker/src/main/java/com/ibm/research/drl/dpt/toolkit/Toolkit.java

```diff
@@ -0,0 +1,242 @@
+/*******************************************************************
+ *                                                                 *
+ * Copyright IBM Corp. 2022                                        *
+ *                                                                 *
+ *******************************************************************/
+package com.ibm.research.drl.dpt.toolkit;
```

Comment:
`dpt.toolkit` looks a bit redundant. maybe we can rename it simply to `dpt.cli` or `dpt.client` ?